src/index.html

```diff
@@ -1,13 +1,539 @@
+              action="#"
+              method="get">
+              <input
+                type="email"
+                class="footer__form form-field"
+                placeholder="Your email"
+                required>
+              <input
+                type="name"
+                class="footer__form form-field"
+                placeholder="Your name"
+                required>
+              <textarea
+                class="
+                  footer__form
+                  form-field
+                  form-field--textarea"
+                placeholder="Your message"
+                required
+                ></textarea>
+              <button type="submit" class="button button--send"
+                >
+                Send
+              </button>
+            </form>
+          </div>
+          <div
+            class="
+              grid__item
+              grid__item--tablet-1-4
+              grid__item--desktop-7-12"
+            >
+            <h3 class="footer__title"
+              >
+              Contact us
+            </h3>
+            <p class="footer__paragraph text--light-grey"
+              >
+              Call us
+            </p>
+            <a href="tel:654 321 987" class="footer__info"
+              >
+              654 321 987
+            </a>
+            <p class="footer__paragraph text--light-grey"
+              >
+              Visit us
+            </p>
+            <a
+              href="https://goo.gl/maps/cDrVBRBeKGvySWtV8"
+              class="footer__info"
+              target="_blank"
+              >
+              2905 West Drive, Buffalo Grove
+            </a>
+            <p
+              class="
+                footer__paragraph
+                text--light-grey
+                footer__paragraph--social"
+              >
+              Our socials
+            </p>
+            <div class="footer__socials">
+              <a
+                href="https://facebook.com"
+                class="icon icon--facebook"
+                target="_blank"
+                >
+              </a>
+              <a
+                href="https://twitter.com"
+                class="icon icon--twitter"
+                target="_blank"
+                >
+              </a>
+              <a
+                href="https://instagram.com"
+                class="icon icon--instagram"
+                target="_blank"
+                >
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="footer__nav">
+          <a href="#" class="logo logo--footer"></a>
+          <div class="menu menu--footer">
+            <a href="#about-us" class="menu__link menu__link--footer"
+              >
+              About us
+            </a>
+            <a href="#home" class="menu__link menu__link--footer"
+              >
+              Home
+            </a>
+            <a href="#work" class="menu__link menu__link--footer"
+              >
```

Comment:
for better readability
```suggestion
            <a 
                href="#work" 
                class="menu__link menu__link--footer"
            >
```

Reply:
I've asked about it and did like as suggested, can you please explain what is wrong?